kodilla-good-patterns/src/main/java/com/kodilla/good/patterns/food2door/data/Order.java

```diff
@@ -0,0 +1,126 @@
+package com.kodilla.good.patterns.food2door.data;
+
+import java.time.LocalDate;
+
+public final class Order {
+    private LocalDate date;
+    private final Company producer;
+    private final Company recipient;
+    private final String productName;
+    private final String measure;
+    private double quantity;
+    private final double price;
+    // derived
+    private final double amount;
+
+    public Order(final LocalDate  date, final Company producer, final Company recipient, final String productName,
+                 final String measure, final double quantity, final double price) {
```

Comment:
this constructor has too many arguments. 
ProductName, measure, price - they can be group together in Product class. Product can also has reference to Producer - and thanks to that we can reduce number of arguments to 4.